spec/classes/corosync_spec.rb

```diff
@@ -65,16 +59,16 @@
 
         it 'configures votequorum' do
           is_expected.to contain_file('/etc/corosync/corosync.conf').with_content(
-            %r{nodelist}
+            %r{nodelist\s*\{}
```

Comment:
I'm wondering if those additional regular expressions provide any value? Do you think the original tests weren't sufficient enough, or did they cause problems?

Reply:
not really add value, just checking whole line.